Makefile

```diff
@@ -18,7 +18,19 @@ RUNTIME_IMAGE ?= $(BUILDER_IMAGE)-runtime
 
 DEVEL_IMAGE ?= apicast-development
 DEVEL_DOCKERFILE ?= Dockerfile-development
+
 DEVEL_DOCKER_COMPOSE_FILE ?= docker-compose-devel.yml
+DEVEL_DOCKER_COMPOSE_VOLMOUNT_MAC_FILE ?= docker-compose-devel-volmount-mac.yml
+DEVEL_DOCKER_COMPOSE_VOLMOUNT_DEFAULT_FILE ?= docker-compose-devel-volmount-default.yml
+
+os = "$(shell uname -s)"
+
+# if running on Mac
```

Comment:
Wouldn't it be more correct to detect if running on a rhel-based distro? As far as I know, those are the ones that have problems with the `:cached ` and `:delegated` modes.

Reply:
The consistency option (:delegated , :cached) according to this https://docs.docker.com/v17.09/engine/admin/volumes/bind-mounts/#choosing-the--v-or-mount-flag should be ignored on all platforms other than Mac. I did not want to assume that other distros other than are ignoring these options and so may run into this problem.And it is not obvious how to check if a specific non mac platform distro is ignoring these options. Since these options are specific to Mac, I thought it would be more logical to separate them out into a mac specific docker-compose file and use a default default docker-compose files.

Reply:
I see. I was wrong :+1: